python/tk_multi_workfiles/work_files.py

```diff
@@ -96,7 +98,7 @@ def wrapper(*args, **kwargs):
         bytes_before = 0
         if sys.platform == "Darwin":
             import resource
-            bytes_before = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss/1024.0/1024.0
+            bytes_before = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss / 1024.0 / 1024.0
```

Comment:
The next three fixes were all that was missing to silence PEP-8, so I figured I might as well do it.